README.md

```diff
@@ -1,4 +1,16 @@
 # Open mSupply
 
-### Client [README](client/README.md)
-### Server [README](server/README.md)
\ No newline at end of file
+This is the home of an open source version of the popular [mSupply](https://msupply.org.nz/) - providing pharmaceutical supply chain management and dispensing, all the way from national warehouses to remote clinics, on servers, laptops and mobile. Used in over 30 countries and for over 20 years and now becoming open source.
```

Comment:
I thought that the initial readme was looking a little stark. I've added some words 🤷

Reply:
btw. I always put one sentence into one line which can make reviewing easier and helps with some diff tools...